src/ext/span.c

```diff
@@ -5,6 +5,7 @@
 #include <unistd.h>
 
 #include "ddtrace.h"
+#include "random.h"
```

Comment:
Why include random.h?

Reply:
This is for the `ddtrace_free_span_id_stack()` call.